telethon/events/callbackquery.py

```diff
@@ -24,11 +24,21 @@ class CallbackQuery(EventBuilder):
             A UTF-8 string can also be given, a regex or a callable. For
             instance, to check against ``'data_1'`` and ``'data_2'`` you
             can use ``re.compile(b'data_')``.
+
+        pattern (`str`, `callable`, `Pattern`, optional):
+            If set, only buttons with payload matching this pattern will be handled.
+            You can specify a regex-like string which will be matched
+            against the message, a callable function that returns ``True``
+            if a message is acceptable, or a compiled regex pattern.
+
     """
     def __init__(
-            self, chats=None, *, blacklist_chats=False, func=None, data=None):
+            self, chats=None, *, blacklist_chats=False, func=None, data=None, pattern=None):
         super().__init__(chats, blacklist_chats=blacklist_chats, func=func)
 
+        if all(x is not None and not x for x in (data, pattern)):
```

Comment:
`all` makes sense when there's a lot of things to compare, but we only have two:
```suggestion
        if data is not None and pattern is not None:
```